packages/one-app-bundler/__tests__/webpack/app/__snapshots__/webpack.client.spec.js.snap

```diff
@@ -22,39 +22,74 @@ Object {
 
 exports[`webpack/app does not transpile node_modules when DANGEROUSLY_DISABLE_DEPENDENCY_TRANSPILATION true 1`] = `
 Object {
-  "exclude": /\\^\\\\/node_modules\\\\/core-js/,
-  "include": Array [
-    "/src",
-  ],
-  "test": /\\\\\\.jsx\\?\\$/,
-  "use": Array [
-    Object {
-      "loader": "babel-loader",
-      "options": Object {
-        "envName": undefined,
-        "extends": "/.babelrc",
+  "enforce": "pre",
```

Comment:
I don't think these should have changed.

Reply:
this is autogenerated :man-shrugging: